src/main/java/com/adzerk/sdk/AdditionalOptions.java

```diff
@@ -44,6 +46,24 @@ public AdditionalOptions apiKey(String apiKey) {
     return this;
   }
 
+  public String[] getDesiredAds() {
+    return this.desiredAds;
+  }
+
+  public AdditionalOptions desiredAds(String desiredAds[]) {
+    this.desiredAds = desiredAds;
+    return this;
+  }
+
+  public Object getDesiredAdMap() {
+    return this.desiredAdMap;
+  }
+
+  public AdditionalOptions desiredAdMap(Object desiredAdMap) {
```

Comment:
`desiredAdMap` should be `Map<String, List<Integer>>` instead of an `Object`.

Reply:
Updated